packages/11ty/_layouts/base.11ty.js

```diff
@@ -27,7 +27,7 @@ module.exports = function(data) {
             aria-expanded="false"
             role="contentinfo"
           >
-            ${this.menu({ imageDir, pageData, pages, publication})}
+            ${this.menu({ pageData, collections, publication})}
```

Comment:
for alphabetic consistency
```suggestion
            ${this.menu({ collections, pageData, publication})}
```